study1/week1/third-maximum-number.java

```diff
@@ -0,0 +1,37 @@
+class Solution {
+    public int thirdMax(int[] nums) {
+        //time complexity O(n)
+        //space complexiti O(1)
+
+        if (nums.length == 0 || nums == null) {
```

Comment:
In the problem statement it says:
```
1 <= nums.length <= 10^4
```
So this condition will never happen.

Reply:
Nonetheless, I thought I should check. thanks for the comment!